data/compatibility/supportStatus.yml

```diff
@@ -7,6 +7,12 @@
   eolDate:
   k8sVersions: ["1.25", "1.26", "1.27", "1.28"]
   testedK8sVersions: ["1.21", "1.22", "1.23", "1.24"]
+- version: "1.19"
+  supported: "Yes"
+  releaseDate: "Sept 1, 2023"
+  eolDate: "~March 2024 (Expected)"
+  k8sVersions: ["1.25", "1.26", "1.27", "1.28"]
+  testedK8sVersions: ["1.21", "1.22", "1.23", "1.24"]
```

Comment:
Verified Prow is running Integration tests for k8s 1.21-1.28.

Reply:
My earlier comment was just to say that we are testing the union of the two lines. And I agree we would say we support the current k8s 1.28.x release and the 3 prior.